Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: all clean plugin docker-install
+.SUFFIXES:
+
+version := $(shell grep '^    <version>.*</version>' pom.xml | cut -d '>' -f 2 | cut -d '<' -f 1)
```

Comment:
Heh, I thought that I could do shorter than this with `sed`, but it turns out two `cut`s is shorter.